FlowCrypt/Functionality/DataManager/SessionService.swift

```diff
@@ -160,9 +156,7 @@ extension SessionService: SessionServiceType {
             logger.logWarning("currentAuthType is not resolved")
         }
         do {
-            for storage in self.storages {
-                try storage.logOutUser(email: user.email)
-            }
+            try encryptedStorage.logOutUser(email: user.email)
```

Comment:
@ioanmo226 after this change `LocalStorage.cleanup` won't be called on log out.

What I meant - we should just remove `try passPhraseStorage.logOutUser(email: email)` from `logOutUser` method of `LocalStorage` and call it in `SessionService` instead (as it's better place to store `passPhraseStorage` dependency)

Reply:
Ohh, I missed it. Thanks for your notice

Reply:
And we still need to add `try passPhraseStorage.logOutUser(email: email)` here 🙂
(which was previously called in `LocalStorage.logOutUser`)

Reply:
Yeah. Right 👍